src/integrationtests/java/com/aws/greengrass/integrationtests/logmanager/SpaceManagementTest.java

```diff
@@ -165,8 +173,6 @@ void setupKernel(Path storeDirectory)
         kernel.launch();
         assertTrue(logManagerRunning.await(10, TimeUnit.SECONDS));
         logManagerService.getUploader().setCloudWatchLogsClient(cloudWatchLogsClient);
-        logManagerService.lastComponentUploadedLogFileInstantMap.put(componentName,
```

Comment:
This was wrong. We mocked the lastUploadedTime for the integration tests effectively forcing the code to behave differently